CPAC/resources/configs/pipeline_config_blank.yml

```diff
@@ -1060,6 +1060,7 @@ functional_preproc:
       fmap_skullstrip_AFNI_threshold: 0.6
 
   func_masking:
+    run: On
```

Comment:
The options in the blank config should all be Off, so that when users import from the blank they only have to list what they want to run.